views/form/editor.html

```diff
@@ -30,7 +32,12 @@
 'formSaved': __('Form saved.'),
 'addFormWidgetHint': __('Add this form to your sidebar or footer on the [link]Widgets page[/link].'),
 'addFormShortcodeHint': __('Copy and paste this [link]shortcode[/link] on to a post or page.'),
-'addFormPhpIframeHint': __('%sPHP%s and %siFrame%s versions are also available.')
+'addFormPhpIframeHint': __('%sPHP%s and %siFrame%s versions are also available.'),
+'settingsListLabel': __('This form adds the subscribers to these lists:'),
+'settingsAfterSubmit': __('After submit...'),
```

Comment:
I think the interpunction should not be part of the translations. That way we couldn't be messed up by translators. And you should use … instead of ...